src/Components/Movies.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import React from 'react';
+
+class Movies extends React.Component {
+
+    state = { movies: [], selectedMovieURL: "", selectedMovie: {} }
+
+    fetchGhibliMovies = async () => {
+        try {
+            const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+            this.setState({ movies: res.data })
+        } catch (error) {
+            this.setState({ movies: [] })
+        }
+    }
+
+    componentDidMount = () => {
```

Comment:
componentDidMount should not be assigned an anonymous function. All lifecycle methods should be written as such: componentDidMount() {}